architecture/core_concepts/routes.adoc

```diff
@@ -212,25 +212,28 @@ $ oc set env dc/router HAPROXY_ROUTER_SYSLOG_ADDRESS=127.0.0.1 HAPROXY_ROUTER_LO
 |`*PROJECT_LABELS*` |  | A label selector to apply to projects to watch, emtpy means all.
 |`*RELOAD_SCRIPT*` |  | The path to the reload script to use to reload the router.
 |`*ROUTER_ALLOWED_DOMAINS*` | | A comma-separated list of domains that the host name in a route can only be part of. Any subdomain in the domain can be used. Option `ROUTER_DENIED_DOMAINS` overrides any values given in this option. If set, everything outside of the allowed domains will be rejected.
-|`*ROUTER_BACKEND_CHECK_INTERVAL*` | 5000ms | Length of time between subsequent "liveness" checks on backends.
+|`*ROUTER_BACKEND_CHECK_INTERVAL*` | 5000ms | Length of time between subsequent "liveness" checks on backends. [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
```

Comment:
Can we say this in a more humane way?  Dumping the regex in is not the most inteligable thing.  Perhaps say "go interval format" or "interval format" and describe it in a new subsection?  Also, I think it takes bare numbers and assumes they are seconds...